doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -74,6 +74,9 @@ Building OpenCV from Source Using CMake
     -   run: “Configure”
     -   run: “Generate”
 
+@note
```

Comment:
You have to add 4 spaces before `@note` to put this section into numbered list:
```
    @note
    Use `cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local ..` , without spaces after -D if the above example doesn't work.
```
otherwise it will break the numbering:
![image](https://cloud.githubusercontent.com/assets/3304494/21050624/c5ed4516-be2c-11e6-9537-c4d8ee17b3b3.png)